src/games/strategy/triplea/TripleA.java

```diff
@@ -43,15 +40,17 @@
 import games.strategy.triplea.ui.display.TripleaDisplay;
 
 public class TripleA extends AbstractGameLoader implements IGameLoader {
-  // compatible with 0.9.0.2 saved games
   private static final long serialVersionUID = -8374315848374732436L;
   public static final String HUMAN_PLAYER_TYPE = "Human";
   public static final String WEAK_COMPUTER_PLAYER_TYPE = "Easy (AI)";
   public static final String FAST_COMPUTER_PLAYER_TYPE = "Fast (AI)";
   public static final String PRO_COMPUTER_PLAYER_TYPE = "Hard (AI)";
   public static final String DOESNOTHINGAI_COMPUTER_PLAYER_TYPE = "Does Nothing (AI)";
-  // public static final String NONE = "None (AI)";
-  protected transient ITripleaDisplay m_display;
+  protected transient ITripleaDisplay display;
+
+  public TripleA() {
```

Comment:
opportunistic improvement. For a longer class, at least one explicit constructor is useful to make the constructors easy to find (rather than not finding them and deducing that there are none)